resources/test/dogu-sort-004.json

```diff
@@ -0,0 +1,6609 @@
+[
```

Comment:
Nice testdata but i think only using necessary attributes makes it easier to perceive the dogus to test.